cypress/integration/views/tags.view.ts

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 export const createTagButton = "Create tag";
-export const createTagtypeButton = "Create tag type";
+export const createTagtypeButton = "Create tag category";
```

Comment:
LGTM!

**nitpick:** Maybe the constant name should be renamed too to avoid confusion

Reply:
Thanks for pointing this out @abrugaro . Done

Reply:
Nice, but in that case we also should rename all usages.

![image](https://user-images.githubusercontent.com/117646518/223070738-08d63853-390b-4021-a12f-405b2eacbced.png)

Reply:
Yes, making changes :)

Reply:
Looks perfect now!!!